File: app/src/main/java/com/kanzankazu/itungitungan/view/util/AppUtil.java
package com.kanzankazu.itungitungan.view.util;

public class AppUtil {
    /**
     * @param b
     * @return 1=true, 0=false
     */
    public static int saveModelBoolInt(boolean b) {
        if (!b) {
            return 0;
        } else {
            return 1;
        }
    }
}

File: app/src/main/java/com/kanzankazu/itungitungan/util/AppUtil.kt
package com.kanzankazu.itungitungan.util

import android.view.View
import android.widget.TextView

object AppUtil {
    fun checkIsStringNotEmptyVisibilityView(string: String, textView: TextView): Boolean {
        return if (string.isNotEmpty()) {
            textView.text = string
            textView.visibility = View.VISIBLE
            true
        } else {
            textView.visibility = View.GONE
            false
        }
    }
}
